tmail-backend/jmap/extensions-cassandra/src/main/scala/com/linagora/tmail/james/jmap/publicAsset/CassandraPublicAssetDAO.scala

```diff
@@ -41,7 +42,8 @@ object CassandraPublicAssetTable {
       .withColumn(BLOB_ID, TEXT)
       .withColumn(SIZE, BIGINT)
       .withColumn(CONTENT_TYPE, TEXT)
-      .withColumn(IDENTITY_IDS, frozenSetOf(DataTypes.UUID)))
+      .withColumn(IDENTITY_IDS, frozenSetOf(DataTypes.UUID))
+      .withColumn(CREATE_DATE, DataTypes.TIMESTAMP))
```

Comment:
Technically adding this new column is unecessary: to order items by time, a TIMEUUID like assetId is supposed to be enough. We could even ask cassandra to sort the itens itself and return the oder first.